var/spack/repos/builtin/packages/rocm-opencl/package.py

```diff
@@ -27,7 +37,8 @@ def flag_handler(self, name, flags):
         # commented out. So instead we provide
         # all the includes...
 
-        if self.spec.satisfies('@3.5.0') and name in ('cflags', 'cxxflags'):
+        if (self.spec.satisfies('@3.5.0') or self.spec.satisfies('@3.7.0')) \
```

Comment:
Do we even need to bother checking the version here since those are the only versions in the package?

Reply:
yes, we can remove this